README.adoc

```diff
@@ -21,50 +21,50 @@ permalink: /guides/jenkins-integration-with-kabanero/
 :page-description: Learn how to use Tekton to build an image and how to use Jenkins to deploy an application that uses that image.
 :page-tags: ['Tekton', 'Jenkins', 'CI/CD', 'Pipeline', 'Kabanero']
 :page-guide-category: collections
-= Jenkins integration with Kabanero
+= Jenkins integration
 
-Kabanero delivers a modern DevOps toolchain with pre-built Tekton pipelines. The Tekton open source project provides Kubernetes-style resources for declaring CI/CD-style pipelines. While it is advantageous to run both CI and CD with Tekton to leverage the power of Kubernetes, some might want to reuse Jenkins assets or skills.
+The product delivers a modern DevOps toolchain with pre-built pipelines. Kubernetes-style resources are provided for declaring CI/CD-style pipelines. While it is advantageous to run both CI and CD style pipelines leverage the power of Kubernetes, some might want to reuse Jenkins assets or skills.
```

Comment:
both CI and CD style pipelines leverage . <- there seems to be a missing word in there somewhere.

Reply:
I added "to" back in.

Reply:
need another "to"
While it is advantageous to run both CI and CD style pipelines *to* leverage